reana_job_controller/job_manager.py

```diff
@@ -98,7 +101,7 @@ def create_job_in_db(self, backend_job_id):
             backend_job_id=backend_job_id,
             workflow_uuid=self.workflow_uuid,
             status=JobStatus.created.name,
```

Comment:
This line should be:

```
status=JobStatus.created.job_name,
```

Otherwise one cannot retrieve job status:

```
Workflow status could not be retrieved:
'Job' object has no attribute 'name'
```

Reply:
Actually, the problem was in workflow-controller - https://github.com/reanahub/reana-workflow-controller/pull/224/files#diff-3ee9faf1efe690830f3d86cb5d3cf590

Thanks for spotting it!